executor/write.go

```diff
@@ -500,15 +502,26 @@ func (e *LoadDataInfo) insertData(cols []string) {
 	}
 	row, err := e.insertVal.fillRowData(e.Table.Cols(), e.row, true)
 	if err != nil {
-		log.Warnf("Load Data: insert data:%v failed:%v", e.row, errors.ErrorStack(err))
+		warnLog := fmt.Sprintf("Load Data: insert data:%v failed:%v", e.row, errors.ErrorStack(err))
+		e.insertVal.handleLoadDataWarnings(err, warnLog)
 		return
 	}
 	_, err = e.Table.AddRecord(e.insertVal.ctx, row)
 	if err != nil {
-		log.Warnf("Load Data: insert data:%v failed:%v", row, errors.ErrorStack(err))
+		warnLog := fmt.Sprintf("Load Data: insert data:%v failed:%v", row, errors.ErrorStack(err))
+		e.insertVal.handleLoadDataWarnings(err, warnLog)
 	}
 }
 
+func (e *InsertValues) handleLoadDataWarnings(err error, errMsg string) {
```

Comment:
`errMsg` is not the message of `err`, but a message you will print as log. It will mislead the readers.